src/js/map/maps.js

```diff
@@ -64,8 +80,6 @@ export class MapControl extends Component {
             .map('main-map', mapOptions.leafletOptions)
             .setView([coords["lat"], coords["long"]], coords["zoom"])
 
-        map.attributionControl.addAttribution('&copy; <a href="https://www.openstreetmap.org/#map=6/-28.676/24.677" target="_blank">OpenStreetMap</a> contributors, <a href="https://carto.com/help/working-with-data/attribution/#basemaps" target="_blank">Carto</a>. Powered by <a href="https://openup.org.za/products/wazimap-ng" target="_blank">Wazimap</a>');
```

Comment:
This removed attribution for Open Street Map and Carto, does their Terms of Use allow us not to attribute them?

If this is okay the boundary selection and zoom controls at the bottom right need to be aligned in some way